Sources/SwiftDriver/Driver/Driver.swift

```diff
@@ -957,14 +957,26 @@ public struct Driver {
         outputFileMap: self.outputFileMap,
         moduleName: moduleOutputInfo.name)
 
-    self.apiDescriptorFilePath = try Self.computeSupplementaryOutputPath(
-        &parsedOptions, type: .jsonAPIDescriptor, isOutputOptions: [],
-        outputPath: .emitApiDescriptorPath,
-        compilerOutputType: compilerOutputType,
-        compilerMode: compilerMode,
-        emitModuleSeparately: emitModuleSeparately,
-        outputFileMap: self.outputFileMap,
-        moduleName: moduleOutputInfo.name)
+    let apiDescriptorDirectory: VirtualPath? = if let apiDescriptorDirectoryEnvVar = env["TAPI_SDKDB_OUTPUT_PATH"] {
+        try VirtualPath(path: apiDescriptorDirectoryEnvVar)
+    } else if let ldTraceFileEnvVar = env["LD_TRACE_FILE"] {
+        try VirtualPath(path: ldTraceFileEnvVar).parentDirectory.appending(component: "SDKDB")
+    } else { nil }
+
+    if let apiDescriptorDirectory {
```

Comment:
```suggestion
    if let apiDescriptorDirectory = apiDescriptorDirectory {
```
Again, to avoid a dependency on Swift 5.9 language features.